macros/alter_table_properties.sql

```diff
@@ -884,7 +893,6 @@ ALTER TABLE cow_protocol_ethereum.batches SET TBLPROPERTIES ('dune.public'='true
 {% do run_query(transfers_ethereum_erc20) %}
 {% do run_query(seaport_ethereum_view_transactions) %}
 {% do run_query(seaport_ethereum_transfers) %}
-{% do run_query(uniswap_trades) %}
```

Comment:
The run_query(uniswap_trades) was here twice. Let me know if you want to remove that in a separate PR.

@soispoke

Reply:
please see this very recent PR to get rid of the alter_table_properties macro #1592

Reply:
Thanks! I've added the post hook to the model I'm adding. 

Not sure if I need to do anything else.